testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py

```diff
@@ -610,19 +609,19 @@ def test_delta_sync_dbWorker(params_from_base_test_setup, num_of_docs, replicati
                                               replication_type="pull")
     replicator.stop(repl)
 
-    # 4. update docs in SGW/CBL
+    # 3. update docs in SGW/CBL
     sg_docs = sg_client.get_all_docs(url=sg_admin_url, db=sg_db, include_docs=True)["rows"]
     update_docs(replication_type, cbl_db, db, sg_client, sg_docs, sg_url, sg_db, number_of_updates, session, channels)
-    sg_client.add_docs(url=sg_url, db=sg_db, number=100, id_prefix="db_worker2", channels=channels, auth=session)
+    sg_client.add_docs(url=sg_url, db=sg_db, number=num_of_docs, id_prefix="db_worker2", channels=channels, auth=session)
 
-    # 5. replicate docs using pull replication
+    # 4. Do push/pull replication
```

Comment:
is below replicator supposed to be push/pull? if yes, then please correct it, as replication type is only pull.

Reply:
both push and pull, that is why I put slash. I am passing parameters one with push , one with pull